src/ol_infrastructure/infrastructure/consul/__main__.py

```diff
@@ -25,7 +25,7 @@
 
 stack_info = parse_stack()  # needed without salt?
 env_config = Config("environment")
-consul_config = Config("consul")
+consul_config = Config("consul")  # where is this located?
```

Comment:
This is a Pulumi `Config` object that pulls data from the stack settings which is a YAML file that neighbors this one. The config object will read values of the form `consul:foovar: foovalue` so that you can then do `consul_config.get('foovar')` to retrieve `foovalue`. (https://www.pulumi.com/docs/intro/concepts/config/#code)